libbeat/logp/file_rotator.go

```diff
@@ -67,7 +67,6 @@ func (rotator *FileRotator) WriteLine(line []byte) error {
 		}
 	}
 
-	line = append(line, '\n')
```

Comment:
A quick test comparing json.Marshal() to common.JSONEncode() would be interesting to see if they produce the same output or a if one has a newline.

Is this code also used by the logp to write files? Does this affect it?